src/plugins/intel_gpu/tests/unit/transformations/convert_convolution_test.cpp

```diff
@@ -486,27 +423,6 @@ TEST_F(TransformationTestsF, ConvertGroupConvolutionToInternal_5) {
 
         model = std::make_shared<ov::Model>(ov::NodeVector{ conv }, ov::ParameterVector{ input });
         manager.register_pass<ConvertConvolutionToInternal>();
-    }
```

Comment:
Do these tests matter with obvious model == model_ref?

Reply:
I think so
We apply `ConvertConvolutionToInternal` pass to the original model, and expect that it's unchanged after that pass